libcaf_net/src/detail/rfc6455.cpp

```diff
@@ -84,6 +84,10 @@ ptrdiff_t rfc6455::decode_header(const_byte_span data, header& hdr) {
   bool masked = (byte2 & 0x80) != 0;
   auto len_field = byte2 & 0x7F;
   size_t header_length;
+  // control frames can only have payload up to 125 bytes
+  if (rfc6455::is_control_frame(hdr.opcode) && len_field > 125) {
+    return -1;
+  }
```

Comment:
@Neverlord  This logic could also go to framing::consume? Only thing I think it's important is to detect it early and stop reading the payload.

Reply:
Intuitively it makes sense to have this here to have it centrally. But no preference either way.